### PR TITLE
new package: python-discogs_client-2.2.0

### PR DESCRIPTION
**Pull request by tonkazoid**
Optional dependency for the beets package

### CODE REVIEW

File: srcpkgs/discogs_client/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+noarch=yes
+build_style="python-module"
+hostmakedepends="python-setuptools"
+depends="python-requests python-oauthlib"
+pycompile_module="discogs_client"
+short_desc="Official Discogs API client for Python"
+maintainer="Georg Schabel <gescha@posteo.de>"
+homepage="https://github.com/discogs/discogs_client"
+license="Discogs"
```
Comment (Duncaen) [inline srcpkgs/discogs_client/template]: It looks like 2-clause-BSD

File: srcpkgs/discogs_client/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+noarch=yes
+build_style="python-module"
+hostmakedepends="python-setuptools"
+depends="python-requests python-oauthlib"
+pycompile_module="discogs_client"
+short_desc="Official Discogs API client for Python"
+maintainer="Georg Schabel <gescha@posteo.de>"
+homepage="https://github.com/discogs/discogs_client"
+license="BSD-2"
+distfiles="https://github.com/discogs/${pkgname}/archive/v${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/discogs_client/template]: Can we use `${homepage}` here?

**Comment by Vaelatern** [comment]
Why should we include this when it seems to be just a pip module best installed from pip?

**Comment by Vaelatern** [comment]
Can we get a python 3 subpackage?

**Comment by Vaelatern** [comment]
Should be named like the python module it is.

**Comment by tonkazoid** [comment]
Something like `discogs_client-python discogs_client-python3.4`?

**Comment by Vaelatern** [comment]
Sure. But first try building it with python3.4 at all

**Comment by Vaelatern** [comment]
Thank you for your contribution!

**Pull request closed without merging**